p2p/discover/table_test.go

```diff
@@ -144,6 +144,39 @@ func fillBucket(tab *Table, ld int) (last *Node) {
 	return b.entries[bucketSize-1]
 }
 
+// This checks that the table-wide IP limit is applied correctly.
+func TestTable_IPLimit(t *testing.T) {
```

Comment:
It is not clear to me what the difference should be between this test and `TestTable_BucketIPLimit`..? 

FYI My proposed changes from https://github.com/whilei/go-ethereum/commit/dfbe2610d7091bd5cecc60c5177f7a3751fc9e01 fix the [`panic` currently causing these tests to fail](https://ci.appveyor.com/project/splix/go-ethereum/build/1.0.1662#L417), but still this test fails because table ip limit is exceeded.